client/state/selectors/is-site-vip.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import { getRawSite } from 'state/sites/selectors';
+
+/**
+ * Returns true if the site is VIP
+ *
+ * If the site is missing returns null.
+ *
+ * @param  {Object}   state  Global state tree
+ * @param  {Number}   siteId Site ID
+ * @return {?Boolean}        Whether site is VIP
+ */
+export default function isSiteVip( state, siteId ) {
```

Comment:
Half of the selectors are `isSiteSomething` and half are `isSomethingSite`. I don’t have a strong opinion on the readability, but I wouldn’t use consistency as an argument.